pt-BR/mail/chrome/messenger/preferences/receipts.dtd

```diff
@@ -1,5 +1,5 @@
 <!ENTITY dialog.title                  "Confirmações de leitura">
-<!ENTITY receiptSettings.label         "Return Receipts">
+<!ENTITY receiptSettings.label         "Confirmações de leitura">
 <!ENTITY requestReceipt.label          "Sempre solicitar uma confirmação de leitura">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY requestReceipt.label          "Ao enviar mensagens, sempre solicitar uma confirmação de entrega">